docs/source/install/updating_installation.rst

```diff
@@ -103,4 +97,10 @@ To find the update instructions for older versions of AiiDA follow the following
 .. _0.6.* Django: http://aiida-core.readthedocs.io/en/v0.7.0/installation.html#updating-from-0-6-0-django-to-0-7-0-django
 .. _0.6.* SqlAlchemy:   http://aiida-core.readthedocs.io/en/v0.7.0/installation.html#updating-from-0-6-0-django-to-0-7-0-sqlalchemy
 .. _0.5.* Django: http://aiida-core.readthedocs.io/en/v0.7.0/installation.html#updating-from-0-5-0-to-0-6-0
-.. _0.4.* Django: http://aiida-core.readthedocs.io/en/v0.5.0/installation.html#updating-from-0-4-1-to-0-5-0
\ No newline at end of file
+.. _0.4.* Django: http://aiida-core.readthedocs.io/en/v0.5.0/installation.html#updating-from-0-4-1-to-0-5-0
```

Comment:
Please change the first paragraph in `updating for aiida developers` to tell them to use `verdi --version` to get the current version number, instead of going into a shell.

Reply:
Will do, and I will generally merge the "for developers" section with the regular upgrade instructions.
Is the sentence " Each version increase may come with its own necessary migrations and you should only ever update the version by one at a time." actually true?
It should be fine to skip aiida versions, right?